pages/src/source/buildings/school.md

```diff
@@ -6,9 +6,8 @@ layout: default
 
 {% capture content %}
 ### Note: The School cannot be built until you finish the research in the [University](../../source/buildings/university).
-<br>
 
-The School is where the Teacher will level up the Pupils' (children) skills. Paper will increase the leveling speed of the Pupils. 
+The School is where the Teacher will level up the Pupils' (children) Intelligence skill. Paper will increase the leveling speed of the Pupils. 
```

Comment:
"School" is not really a special name in the regard that it should be capitalized, please make it lowercase "school", same with "Teacher" and "Pupils'"

Reply:
I thought the hut and worker names were capitalized intentionally, so I've been keeping them that way.

Reply:
I thought they were too TBH - made sense to me that the workers and hut names were capitalised

Reply:
Workers and hut names are capitalized intentionally.